litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddProbe/ProbeDetails.tsx

```diff
@@ -165,7 +165,7 @@ const ProbeDetails: React.FC<ProbeDetailsProps> = ({
           </div>
           <div className={classes.formField}>
             <InputLabel className={classes.formLabel} htmlFor="responseTimeout">
-              Response Timeout
+              Response Timeout(ms)
```

Comment:
any specific use-case for not using translation here? @saswatamcode

Reply:
No. Adding it into translation file!